src/resmom/start_exec.c

```diff
@@ -4254,7 +4234,7 @@ finish_exec(job *pjob)
 	endpwent();
 
 	job_has_executable = 0;
-	if (pjob->ji_wattr[(int)JOB_ATR_executable].at_flags & ATR_VFLAG_SET) {
+	if (is_job_attr_set(pjob, JOB_ATR_executable)) {
```

Comment:
I think there are still few place where is_job_attr_set can be used, for example:
```
./src/resmom/start_exec.c:308:	if ((pjob->ji_wattr[(int)JOB_ATR_egroup].at_flags &
		(ATR_VFLAG_SET | ATR_VFLAG_DEFLT)) == ATR_VFLAG_SET) {
./src/resmom/start_exec.c:4050:		if (!(pjob->ji_wattr[(int)JOB_ATR_executable].at_flags &
			ATR_VFLAG_SET)) {
./src/resmom/start_exec.c:4416:	if ((pjob->ji_wattr[(int)JOB_ATR_executable].at_flags &
		ATR_VFLAG_SET) && (sandbox_private)) {
./src/resmom/start_exec.c:6536:	if ((pjob->ji_wattr[(int)JOB_ATR_interactive].at_flags&ATR_VFLAG_SET) &&
./src/resmom/start_exec.c:6614:		if ((pjob->ji_wattr[(int)JOB_ATR_sandbox].at_flags&ATR_VFLAG_SET) &&
./src/resmom/start_exec.c:6625:		if ((pjob->ji_wattr[(int)JOB_ATR_sandbox].at_flags&ATR_VFLAG_SET) &&
./src/resmom/catch_child.c:1276:		if( pjob->ji_wattr[(int)JOB_ATR_Comment].at_flags & \
							ATR_VFLAG_SET) {
./src/resmom/catch_child.c:1381:		if (pjob->ji_wattr[(int)JOB_ATR_Cookie].at_flags &
			ATR_VFLAG_SET) {
./src/resmom/mom_comm.c:1040:	if ( !(pjob->ji_wattr[(int)JOB_ATR_Cookie].at_flags & \
				ATR_VFLAG_SET) ) {
./src/resmom/mom_comm.c:3344:				if (pjob->ji_wattr[(int)JOB_ATR_umask].at_flags&ATR_VFLAG_SET) {
./src/resmom/mom_main.c:3069:		if ((pjob->ji_wattr[(int)JOB_ATR_sandbox].at_flags &
			ATR_VFLAG_SET) &&
./src/resmom/mom_main.c:6890:	if ((pjob->ji_wattr[(int)JOB_ATR_interactive].at_flags&ATR_VFLAG_SET) &&
./src/server/svr_func.c:5565:	if (pjob->ji_wattr[(int)JOB_ATR_prov_vnode].at_flags &
		ATR_VFLAG_SET) {
./src/server/req_quejob.c:910:		if (!(pj->ji_wattr[(int)JOB_ATR_job_owner].at_flags &
			ATR_VFLAG_SET)) {
./src/server/req_quejob.c:950:		if ((pj->ji_wattr[(int)JOB_ATR_cred].at_flags &
			ATR_VFLAG_SET) == 0 ||
./src/server/req_quejob.c:1166:			if ((pjob->ji_wattr[(int)JOB_ATR_block].at_flags &
				ATR_VFLAG_SET) == 0)
./src/server/req_quejob.c:1300:		if ((pj->ji_wattr[(int)JOB_ATR_euser].at_flags & \
							ATR_VFLAG_SET) &&
./src/server/req_modify.c:341:	if (mod_project && (pjob->ji_wattr[(int)JOB_ATR_project].at_flags & \
							ATR_VFLAG_SET)) {
./src/server/svr_jobfunc.c:407:			if (pjob->ji_wattr[(int)JOB_ATR_depend].at_flags&ATR_VFLAG_SET) {
./src/server/svr_jobfunc.c:416:		if (((pjob->ji_wattr[(int)JOB_ATR_etime].at_flags &
			ATR_VFLAG_SET) == 0) &&
./src/server/svr_jobfunc.c:1089:	if (!(pjob->ji_wattr[(int)JOB_ATR_euser].at_flags &
		ATR_VFLAG_SET) ||
./src/server/svr_jobfunc.c:1091:		!(pjob->ji_wattr[(int)JOB_ATR_egroup].at_flags &
		ATR_VFLAG_SET)) {
./src/server/svr_jobfunc.c:1204:		if (((pjob->ji_wattr[(int)JOB_ATR_cred].at_flags &
			ATR_VFLAG_SET) == 0 ||
./src/server/svr_jobfunc.c:4664:			else if ((pjob->ji_wattr[(int) JOB_ATR_exit_status].at_flags) &
				 ATR_VFLAG_SET) {
./src/server/svr_jobfunc.c:5082:			} else if ((pjob->ji_wattr[(int)JOB_ATR_exit_status].at_flags) &
				ATR_VFLAG_SET) {
./src/server/req_runjob.c:1209:	if (jobp->ji_wattr[(int)JOB_ATR_depend].at_flags&ATR_VFLAG_SET)
./src/server/job_func.c:2233:		return pjob->ji_wattr[attr_idx].at_flags & ATR_VFLAG_SET;
./src/server/job_func.c:2243:		pjob->ji_wattr[attr_idx].at_flags &= ~ATR_VFLAG_SET;
./src/server/pbsd_main.c:546:			if (((pjob->ji_wattr[(int)JOB_ATR_exec_vnode].at_flags &
				ATR_VFLAG_SET) != 0) &&
./src/server/pbsd_main.c:1800:			if ((pjob->ji_wattr[(int)JOB_ATR_exec_vnode].at_flags &
				ATR_VFLAG_SET) != 0) {
./src/server/pbsd_init.c:1410:				if (pjob->ji_wattr[(int)JOB_ATR_prov_vnode].at_flags &
					ATR_VFLAG_SET) {
./src/server/array_func.c:512:			if ((pjob->ji_wattr[(int)JOB_ATR_stageout_status].at_flags) & ATR_VFLAG_SET) {
./src/server/array_func.c:516:			if ((pjob->ji_wattr[(int)JOB_ATR_exit_status].at_flags) & ATR_VFLAG_SET) {
./src/server/req_register.c:157:	if (pjob->ji_wattr[(int)JOB_ATR_depend].at_flags&ATR_VFLAG_SET)
./src/server/req_jobobit.c:424:	if ((pjob->ji_wattr[(int)JOB_ATR_exit_status].at_flags) &
					ATR_VFLAG_SET) {
```